packages/flame/lib/src/cache/images.dart

```diff
@@ -1,6 +1,5 @@
 import 'dart:async';
 import 'dart:convert' show base64, json;
-import 'dart:typed_data';
```

Comment:
Don't we have to bump the min version of Flutter in the pubspec if we remove this?

Reply:
I'm not sure -- it was just an analyzer warning I fixed.
I can bump the version if you want though, just lmk which.

Reply:
It's an analyzer warning in the current version, but would be an error when the line is removed when running with an older version of Flutter right?

Reply:
Currently CI builds with the latest stable to make sure that we support the latest stable (which was why we got these analyze issues now). I think it would be fine to just bump the Flutter version in the pubspecs? I don't think there are any blockers for anyone to move to the latest stable flutter version.

Reply:
So I changed my channel to stable and upgraded. Latest version is 3.3.0. Ours is currently 3.0.0. So I am updating all references.
<img width="270" alt="image" src="https://user-images.githubusercontent.com/882703/187963498-61749e3e-66e2-4031-a99f-e237104fc05c.png">